Riot/Modules/Authentication/AuthenticationCoordinator.swift

```diff
@@ -91,6 +94,17 @@ final class AuthenticationCoordinator: NSObject, AuthenticationCoordinatorProtoc
         authenticationViewController.continueSSOLogin(withToken: loginToken, txnId: transactionID)
     }
     
+    func allowScreenPresentation() {
```

Comment:
This is a weird name for a method, it needs a better verb as it's doing more than just allowing a screen presentation. 
```

/// Indicates to the coordinator to display any pending screens if it was created with
/// the `canPresentAdditionalScreens` parameter set to `false`
```
Based on the docs, we can go with `presentPendingScreensIfNecessary`.

Also, perhaps we shouldn't show a loading animation if `isWaitingToPresentCompleteSecurity` is `false`? 🤔

Reply:
Thank you, I didn't massively like this, your suggestion is much better 🙌

Reply:
Re the animation, the intent was to keep the existing behaviour where the security screen is shown as a modal on top of the spinner. This way you won't see the last screen in the onboarding process when the modal is dismissed. I will make sure to raise this/iterate on it before releasing the feature.